### PR TITLE
chore: Enable ORA Grading MFE for sandboxes

### PR DESCRIPTION
**Pull request by jnlapierre**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A SRE team member has approved the PR if it is code shared across multiple services and you don't own all of the services.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a SRE ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/spaces/SRE/pages/28967861/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).
  - [ ] Think about how this change will affect Open edX operators.  Have you updated the wiki page for the next Open edX release?

### CODE REVIEW

File: playbooks/edx_continuous_integration.yml [hunk 1]
```diff
@@ -56,6 +56,9 @@
     - role: mfe
       MFE_NAME: learning
       when: LEARNING_MFE_ENABLED
+    - role:mfe
+      MFE_NAME: ora_grading
```
Comment (matthugs) [inline playbooks/edx_continuous_integration.yml]: You'll want the hyphens here: https://github.com/edx/configuration/blob/77a0b37aa6df562087ba6e2f298dad33f46132c4/playbooks/roles/mfe/defaults/main.yml

Comment (matthugs) [reply playbooks/edx_continuous_integration.yml]: and elsewhere where you refer to this role

File: playbooks/edx_continuous_integration.yml [hunk 2]
```diff
@@ -56,6 +56,9 @@
     - role: mfe
       MFE_NAME: learning
       when: LEARNING_MFE_ENABLED
+    - role:mfe
```
Comment (matthugs) [inline playbooks/edx_continuous_integration.yml]:
```suggestion
    - role: mfe
```

File: playbooks/ora_grading.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+- name: Deploy the ORA Grading MFE Frontend
+  hosts: all
+  become: True
+  gather_facts: True
+  vars:
+    ENABLE_NEWRELIC: False
+    CLUSTER_NAME: 'ora-grading'
+    ORA-GRADING_ENABLED: True
+    ORA-GRADING_SANDBOX_BUILD: False
```
Comment (matthugs) [inline playbooks/ora_grading.yml]: missed a spot

**Review by matthugs — CHANGES_REQUESTED** [review]

File: playbooks/edx_continuous_integration.yml (hunk 2 above)
Comment (matthugs) [reply playbooks/edx_continuous_integration.yml]: this is the cause for 
```
yaml.scanner.ScannerError: mapping values are not allowed here
  in "./playbooks/edx_continuous_integration.yml", line 60, column 15
```
in https://github.com/edx/configuration/runs/4588077725?check_suite_focus=true

File: playbooks/edx_continuous_integration.yml (hunk 1 above)
Comment (matthugs) [reply playbooks/edx_continuous_integration.yml]: revisiting https://openedx.atlassian.net/wiki/spaces/FEDX/pages/2124677169/How+To+Add+an+MFE+to+a+Sandbox it looks like we want to update the guidance on that subject as well

Comment (jnlapierre) [reply playbooks/edx_continuous_integration.yml]: I have updated the configuration and added a note to the instructions.

File: playbooks/edx_continuous_integration.yml (hunk 2 above)
Comment (jnlapierre) [reply playbooks/edx_continuous_integration.yml]: Missing space has been added.

File: playbooks/ora_grading.yml (hunk 3 above)
Comment (jnlapierre) [reply playbooks/ora_grading.yml]: Oops, face palm, fixed.

File: playbooks/ora_grading.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  become: True
+  gather_facts: True
+  vars:
+    ENABLE_NEWRELIC: False
+    CLUSTER_NAME: 'ora-grading'
+    ORA_GRADING_ENABLED: True
+    ORA_GRADING_SANDBOX_BUILD: False
+  roles:
+    - role: mfe
+      MFE_NAME: ora_grading
```
Comment (matthugs) [inline playbooks/ora_grading.yml]: this is the same situation as https://github.com/openedx/configuration/pull/6641/files#diff-43ed106499a60cebe90128b847e99eed4d09b3720d3ba8231e6e534e3eea0833R60

**Review by matthugs — CHANGES_REQUESTED** [review]

File: playbooks/ora_grading.yml (hunk 4 above)
Comment (jnlapierre) [reply playbooks/ora_grading.yml]: Updated, and this is making more sense to me now.  I suspect it will be easier next time for me.

**Review by matthugs — APPROVED** [review]

**Pull request merged**